index.html

```diff
@@ -9,5 +9,6 @@
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
+        <script src="node_modules/flowbite/dist/flowbite.min.js"></script>
```

Comment:
we shouldn't need this here btw, we shuold be able to import flowbite into the source code where ever we are using it

Reply:
Didn't know about the react components lol. This makes it way easier!

Reply:
Hehe all about that write less code life